lazer/contracts/solana/programs/pyth-lazer-solana-contract/src/lib.rs

```diff
@@ -34,13 +37,13 @@ impl TrustedSignerInfo {
 }
 
 #[account]
-pub struct Storage {
+pub struct StorageV1 {
     pub top_authority: Pubkey,
     pub num_trusted_signers: u8,
     pub trusted_signers: [TrustedSignerInfo; MAX_NUM_TRUSTED_SIGNERS],
 }
 
-impl Storage {
+impl StorageV1 {
```

Comment:
Since we haven't deployed on mainnet yet. Can we just make it a breaking change and not have V1 and V2? It feels weird that we have V1 and V2 even before we have launched on mainnet.

Reply:
I've removed v1 and v2 from the account names and method names, since we cannot easily change them later (that's a breaking change in anchor). I've removed `initialize_v1` method entirely. The migration method uses a temporary `StorageV010` struct to read the previous storage format. We can safely remove that after the migration is done.